modules/ROOT/pages/design-create-publish-api-specs-from-file.adoc

```diff
@@ -0,0 +1,51 @@
+= Create and Publish an API Specification or Fragment from an External File
+
+Use API Designer to create a new API specification or an API fragment from an existing external file. API Designer supports importing from JSON, RAML, or YML files. You can also import API specifications and fragments from an existing project.
```

Comment:
Add zip file format

Reply:
Revised as:

Use API Designer to create a new API specification or an API fragment from an existing external file. API Designer supports importing from JSON, RAML, or YML files. You can also import API specifications and fragments from a ZIP file for an existing project.

Reply:
I don't see the change, just FYI.

Reply:
@Mulesoft-Mysti Verified that these changes are in the branch.